torch/testing/_internal/common_utils.py

```diff
@@ -1770,10 +1773,17 @@ def make_tensor(size, device: torch.device, dtype: torch.dtype, *, low=None, hig
         result = torch.repeat_interleave(result, 2, dim=-1)
         result = result[..., ::2]
 
+    if not include_zero and dtype not in complex_types():
+        if dtype in integral_types() or dtype is torch.bool:
+            replace_with = torch.tensor(1, device=device, dtype=dtype)
+        elif dtype in floating_types_and(torch.half, torch.bfloat16):
+            replace_with = torch.tensor(sys.float_info.epsilon, device=device, dtype=dtype)
+        result[result == 0] = replace_with 
```

Comment:
add an else clause that asserts to be sure no types are missed

Reply:
Thanks, this should be resolved now.